src/test/java/seedu/address/logic/commands/deliverycommand/DeliveryAddCommandIntegrationTest.java

```diff
@@ -33,9 +32,10 @@ public void execute_newDelivery_success() {
                 String.format(DeliveryAddCommand.MESSAGE_SUCCESS, validDelivery), expectedModel);
     }
 
+    /* No such thing as duplicate delivery
     @Test
     public void execute_duplicateDelivery_throwsCommandException() {
         Delivery delivery = model.getDeliveryBook().getDeliveryList().get(0);
         assertCommandFailure(new DeliveryAddCommand(delivery), model, DeliveryAddCommand.MESSAGE_DUPLICATE_DELIVERY);
-    }
+    }*/
```

Comment:
almost LGTM, can you just remove this test case

Reply:
fixed in latest commit